data/style/style.css

```
@@ -1,131 +1,98 @@
-a:link    { font-weight:bold; font-style:normal; color:white;   text-decoration:none; }
-a:visited { font-weight:bold; font-style:normal; color:white;   text-decoration:none; }
-a:focus   { font-weight:bold; font-style:normal; color:white;   text-decoration:underline; }
-a:hover   { font-weight:bold; font-style:normal; color:#6a6868; text-decoration:none; }
-
-body{
-  background-color:#5a5858;
-  font-family:trebuchet ms,bitstream vera sans,verdana,helvetica,arial,sans-serif; font-size: 8pt;
+body {
+  background-color: #5a5858;
+  font-family: 'Trebuchet MS', 'Bitstream Vera Sans', Verdana, Helvetica, Arial, sans-serif;
+  font-size: 8pt;
   margin: 0;
   padding: 0;
   border: 0;
 }
 
-body.navbody{
-  font-family:trebuchet ms,bitstream vera sans,verdana,helvetica,arial,sans-serif; font-size: 8pt;
-  background-color:#d2d0d0;
-  margin: 0;
-  padding: 0;
-  border: 0;
+header {
+  max-width: 742px;
+  height: 90px;
+  position: relative;
+  margin: 0 auto;
 }
 
-.title{
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: 50pt;
-  margin-bottom: -5pt;
-  background-color:#5a5858;
-  color:#d2d0d0;
-  padding: 0pt;
-  font-weight:bold;
-  font-size:20pt;
-  width: 746px; /* width of page + padding. */
-}
-.page{
-  background-color:#d2d0d0;
-  padding: 20pt;
-  padding-bottom: 40pt;
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: 0pt;
-  margin-bottom: 0pt;
-  width: 692px;
-  counter-reset: section;
-  border-style:solid;
-  border-width:1px;
-  border-color:#222222;
-}
-.footer{
-  background-image:none;
-  background-color:#5a5858;
-  padding: 0pt;
-  text-align:right;
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: 5pt;
-  margin-bottom: 20pt;
-  width: 692px;
+header h1 {
+  color: #d2d0d0;
+  font-weight: bold;
+  font-size: 26px;
+  position: absolute;
+  bottom: -6px;
```

Changing this to zero will prevent the text cut-off, but add additional space between the text and following container.

```suggestion
  bottom: 0;
```